packages/databyss-ui/modules/Landing/Landing.js

```diff
@@ -145,10 +145,13 @@ class Landing extends React.Component {
       showMotifLinks,
       onMotifLinksChange,
       withToggle,
+      isSource,
     } = this.props
 
     const { headerSticky, inStickyContainer, isViewMobile, width } = this.state
 
+    const sourceHeader = isSource
```

Comment:
Again, let's try to avoid this. If you need to fork the styles or behavior, refactor to `Landing`, `SourceLanding` and `MotifLanding` using a composition pattern (cf `Button`, `PrimaryButton`, etc).